Dockerfile.s390x

```diff
@@ -48,6 +48,21 @@ RUN apt-get update && apt-get install -y \
 	tar \
 	--no-install-recommends
 
+# install seccomp: the version shipped in jessie is too old
+ENV SECCOMP_VERSION 2.3.0
+RUN set -x \
+	&& export SECCOMP_PATH="$(mktemp -d)" \
+	&& curl -fsSL "https://github.com/seccomp/libseccomp/releases/download/v${SECCOMP_VERSION}/libseccomp-${SECCOMP_VERSION}.tar.gz" \
+		| tar -xzC "$SECCOMP_PATH" --strip-components=1 \
+	&& ( \
+		cd "$SECCOMP_PATH" \
+		&& ./configure --prefix=/usr/local \
+		&& make \
+		&& make install \
+		&& ldconfig \
+	) \
+	&& rm -rf "$SECCOMP_PATH"
+
```

Comment:
Hello Chris, many thanks for the hints.

> Also you should take a look at seccomp/libseccomp-golang#7

Do you know the symptom of the bindings problem? Will Docker crash at runtime, when seccomp functionality is used? If this is the case we probably should just remove "seccomp" from runc BUILDTAGS like you have it in Dockerfile.ppc64le:
```
diff --git a/Dockerfile.s390x b/Dockerfile.s390x
index 71bfcaf..ef24ddd 100644
--- a/Dockerfile.s390x
+++ b/Dockerfile.s390x
@@ -193,7 +193,7 @@ RUN set -x \
        && git clone https://github.com/opencontainers/runc.git "$GOPATH/src/github.com/opencontainers/runc" \
        && cd "$GOPATH/src/github.com/opencontainers/runc" \
        && git checkout -q "$RUNC_COMMIT" \
-       && make static BUILDTAGS="seccomp apparmor selinux" \
+       && make static BUILDTAGS="apparmor selinux" \
        && cp runc /usr/local/bin/docker-runc \
        && rm -rf "$GOPATH"
```